lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -1167,6 +1242,34 @@ def rotate_axes(xs, ys, zs, zdir):
         return xs, ys, zs
 
 
+def _set_offsets3d(col_3d, offsets, zdir='z'):
+    """
+    Set the 3d offsets for the collection.
+
```

Comment:
We should explain this somewhat peculiar pattern:

```suggestion

    .. note::
    
        Since 3D collections have no common 3D base class, this function
        factors out the common code for ``set_offests3d()`` methods.

```